src/report_workflow/report_args.py

```diff
@@ -0,0 +1,48 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+#
+# Modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+import argparse
+import logging
+
+from test_workflow.test_kwargs import TestKwargs
+
+
+class ReportArgs:
+    test_run_id: str
+    keep: bool
+    test_manifest_path: str
+    artifact_paths: dict
+    test_type: str
+    logging_level: int
+
+    def __init__(self) -> None:
+        parser = argparse.ArgumentParser(description="Report for test on OpenSearch Bundle")
+        parser.add_argument("test_manifest_path", type=str, help="Specify a test manifest path.")
+        parser.add_argument("-p", "--artifact-paths", nargs='*', action=TestKwargs, default={},
+                            help="Specify aritfacts paths for OpenSearch and OpenSearch Dashboards.")
+        # e.g. --base-path https://ci.opensearch.org/ci/dbc/integ-test/2.7.0/7771/linux/x64/tar/test-results/1234<test-run-id>/integ-test use more to save arguments number
+        parser.add_argument("--base-path", type=str, default="",
+                            help="Specify base paths for the integration test logs.")
+        parser.add_argument("--test-type", type=str, default="integ-test", help="Specify test type of this.")
+        parser.add_argument("--output-path", type=str, help="Specify the path location for the test-run manifest.")
```

Comment:
Default to the current folder from where the command is run. folder

Reply:
It's default to the current folder from the implementation here from `run_test_report` file. Basically this is an optional argument in case user specifies. 
```
test_run = test_run_runner.generate_report(test_run_data, args.output_path or os.getcwd())
```